subsys/usb/class/netusb/function_ecm.c

```diff
@@ -233,41 +233,34 @@ int ecm_send(struct net_pkt *pkt)
 {
```

Comment:
Commit message is missing

Fixes #4637 
Coverity-CID: 178334

Reply:
Added Coverity and bug ID